### DISCUSSION TITLE
Needs more memory

### DISCUSSION THREAD

**Discussion by lfrohling** [whscullin/apple2js#214] [General]
Great idea :) Shame a lot of Apple 2e games don't work as it needs more than 128k of memory to execute programs :(

**Comment by lfrohling** [comment]
Eg. Where in the World is Carmen Sandiago and Where in Time (https://archive.org/details/apple-ii-play-it-by-year-collection-v-0.1)

**Comment by iflan** [comment]
I tested this and the individual 140K disk images work fine as-is. The 800K image does not. It crashes on the first elevator screen. I can't tell if that's a problem with the emulator or the 800K image, but I'm guessing it's the disk image.
